modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/ClassicEditor.js

```diff
@@ -16,13 +16,19 @@ const ClassicEditor = forwardRef(
 			editorConfig,
 			initialToolbarSet = 'simple',
 			name,
+			required,
 			title,
 			...otherProps
 		},
 		ref
 	) => {
 		return (
-			<div className={className} id={`${name}Container`}>
+			<div
+				aria-required={required}
+				className={className}
+				id={`${name}Container`}
+				role="textbox"
+			>
```

Comment:
Where does `aria-required` need to be set? We're setting it in two places in this PR, which seems odd to me.